helm/openshift-gitops-day2/templates/9-forklift-controller.yaml

```
@@ -0,0 +1,18 @@
+{{- with .Values.mtv }}
+
+---
+
+#Creates forklift controller instance for MTV.
+apiVersion: forklift.konveyor.io/v1beta1
+kind: ForkliftController
+metadata:
+  name: forklift-controller
+  namespace: {{ .target_namespace }}
+  annotations:
+    argocd.argoproj.io/sync-wave: "4"
```

I'm not sure what provides the Forklift controller crd. If it's the operator itself we need to add the flag to tell argo to ignore schema validation failures since the crd won't exist yet. that's this one:

```suggestion
    argocd.argoproj.io/sync-wave: "4"
    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
```

---

Ah, good to know!